src/modsim2/similarity/mmd.py

```diff
@@ -46,6 +54,21 @@ def mmd(
     kernel_fn = MMD_KERNEL_DICT[kernel_name]
 
     # Compute MMD components
+    logging.info(
+        "Computing "
+        + str(N_A)
+        + " by "
+        + str(N_A)
+        + ", "
+        + str(N_B)
+        + " by "
+        + str(N_B)
+        + ", and "
+        + str(N_A)
+        + " by "
+        + str(N_B)
+        + " kernels. This may take some time!"
```

Comment:
Using f-strings would be more canonical, e.g.:

```suggestion
        f"Computing {N_A} by {N_A}, {N_B} by {N_B}, and {N_A} by {N_B} kernels. "
        "This may take some time!"
```

(not checked line length, single line may be ok but can split across multiple lines like above if not)